City-of-Helsinki/drupal-helfi-platform-config
# UHF-435: Genesys chat

Implements a Genesys TEST chat.

How to test:

- create test environment:
  - `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  - `cd hel-platform`
  - `make new`
- checkout this branch of the `drupal_helfi_platform_config` module: `composer require drupal/helfi_platform_config:dev-UHF-435_genesys_chatbot`
- clear caches: `make drush-cr`
- login to the site as admin
- from the block configuration (`/admin/structure/block`) place the `Genesys Chat` block on the `Attachments` region
- navigate to any content page and see the chat in the lower right corner

NOTE! This is only the test version of the chat just to see how it works with Drupal and our sites.
